src/Testcontainers.Milvus/Testcontainers.Milvus.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>net6.0;netstandard2.0;net462</TargetFrameworks>
```

Comment:
Regarding this, I have a question; maybe you know more about it. Are there significant benefits for users if a library targets the latest frameworks, in addition to .NET Standard, **especially** with transitive dependencies that still target .NET Standard?

I know, as the consumer (library maintainer) of the framework (SDK), I get access to more APIs, etc. But what is exactly the benefit for the Testcontainers user targeting `net6.0;netstandard2.0;net462` instead of `netstandard2.0;netstandard2.1`? I did some research and have not found much information.

Reply:
> Do you mind if I update some parts to align the module with all the others?

Absolutely - please feel free to make any adjustments necessary to align this with the repo standards or anything else!

> But what is exactly the benefit for the Testcontainers user targeting net6.0;netstandard2.0;net462 instead of netstandard2.0;netstandard2.1?

On this... For net462 specifically, the .NET Framework build of the Milvus client uses the .NET Framework build of Grpc.Net.Client, because that automatically uses WinHttpHandler; if you target netstandard2.0, that helpful feature isn't there. tl;dr it makes the experience easier on .NET Framework, where some extra config/wiring up  is necessary.

For net6.0, I think the main reason there's that TFM is to not bring in various dependencies that are necessarily on .NET Standard (e.g. Microsoft.Bcl.HashCode) but are already included in the box in net6.0. So you get cleaner dependency graph etc. Of course, in theory there may also be new APIs exposed that depend on newer .NET functionality, but specifically for Milvus.Client there aren't any IIRC.

Hopefully that all makes sense - let me know if you have any further questions!

Reply:
> I think the main reason there's that TFM is to not bring in various dependencies that are necessarily on .NET Standard (e.g. Microsoft.Bcl.HashCode) but are already included in the box in net6.0.

I see, I didn't know that the TFM uses its own implementation instead of the one from the dependency. Then it definitely makes sense to target TC for other TFMs too. Thanks 🙏.

Reply:
Yeah, this is basically for functionality that got added at some point (e.g. to net6.0), but a compat nuget was released to also support older versions. It's no big deal if you use netstandard2.0 and get that package, but slightly cleaner without it.